packages/core/src/checkout/checkout-action-creator.ts

```diff
@@ -117,7 +136,28 @@ export default class CheckoutActionCreator {
                 throw new MissingDataError(MissingDataErrorType.MissingCheckout);
             }
 
-            return this.loadCheckout(checkout.id, options);
+            return this.loadCheckout(checkout.id, options)(store);
+        };
+    }
+
+    private _shouldTransformCustomerAddress(storeConfig: StoreConfig): boolean {
+        return (
+            storeConfig.checkoutSettings.features[
+                'CHECKOUT-8183.set_shouldSaveAddress_false_for_existing_address'
+            ] ?? false
```

Comment:
```suggestion
            ] ?? true
```
For the reason that when we remove the experiment value, any existing custom checkout with this SDK change will behave as expected.